### PR TITLE
Include example git hooks

### LINKED ISSUE #8: Consider husky examples
I've just setup https://github.com/typicode/husky with

```shell
npx husky add .husky/post-checkout "npx --no package-changed"
npx husky add .husky/post-merge "npx --no package-changed"
npx husky add .husky/post-rebase "npx --no package-changed"
```

and added `.packagehash` to my `.gitignore` and this seems to work very well.

Would you consider adding something to the readme that would help others figure out how to attain this setup, or something similar (maybe there's a better way?)

### PR DESCRIPTION
**Pull request by nemchik**
This closes https://github.com/thdk/package-changed/issues/8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,20 @@ Options:
...
+
+With Husky installed:
+
+```shell
+npx husky add .husky/post-checkout "npx --no package-changed"
+npx husky add .husky/post-merge "npx --no package-changed"
+npx husky add .husky/post-rebase "npx --no package-changed"
+```
+
+When using this approach, add `.packagehash` to your `.gitignore` so that changes can be properly detected when switching branches.
```
Comment (thdk) [inline README.md:91]:
```suggestion
```
`.packagehash` should be added to `.gitignore` for all possible approaches :)

File: README.md [hunk 2]
```diff
@@ -76,6 +76,20 @@ Options:
   -h, --help  display help for command
 ```
 
+### git hooks
```
Comment (thdk) [inline README.md:79]:
```suggestion
#### Use git hooks to run **package-changed** automatically

```

**Review by thdk — CHANGES_REQUESTED** [review]
Small remarks :)

**Comment by nemchik** [comment]
Updated.

**Review by thdk — APPROVED** [review]
Thanks!

**Pull request merged**